docs/_pages/docs/03-understand/02-robots.md

```diff
@@ -10,4 +10,48 @@ sidebar:
 src: "_pages/docs/03-understand/02-robots.md"
 ---
 
-{% include wip %}
\ No newline at end of file
+{% include wip %}
```

Comment:
We can now get rid of this line, it displays the "Work in progress" notice.

Reply:
Ah I didn't realize what that line intended

Yay! That's awesome, thank you very much :D let me try out this invoice:

lnbc200u1p3s5v32pp5p62jxach58d48u6yg4tt7jms9pvs7m07j4m7dzk2stud6pa62u8qdqu2askcmr9wssx7e3q2dshgmmndp5scqzpgxqyz5vqsp5r7ta253wcwp8usfjdqf5tja474wx877lu49puvqhn5gktud50hvs9qyyssq3cygts9l8r2rhzrv89a6d4wz8ect8296fsp60y32y4k00ghav5fkqm8dvlgme49mjvvcn0a3s8kuw7mzxs4jkltjtn0g4ufxm28c5xqp2txpxu